app/views/cases/data_requests/show.html.slim

```diff
@@ -26,8 +26,11 @@ span.visually-hidden
     .form-group
       label.form-label
         = t('helpers.fieldset.data_request.request_type')
-      p.data_request_request_type
+      p.data_request_request_type style="margin-bottom: 0;"
```

Comment:
It's best to avoid inline styles like this. 
I understand why you have done it, but as an alternative you could use a br within the p tag to add the extra data.